modules/runes/api/httphandler/get_transactions.go

```diff
@@ -125,17 +128,31 @@ func (h *HttpHandler) GetTransactions(ctx *fiber.Ctx) (err error) {
 		}
 	}
 
-	blockHeight := req.BlockHeight
-	// set blockHeight to the latest block height blockHeight, pkScript, and runeId are not provided
-	if blockHeight == 0 && pkScript == nil && runeId == (runes.RuneId{}) {
+	// default to latest block
```

Comment:
You should validate if `toBlock` or `fromBlock` is less than `-1` or make all negative value is equal current block

Reply:
I've another ideas, use python slices logic like if -1 is latestBlock, -2 is latestBlock - 1 and -3 is latestBlock -2